### PR TITLE
chore: release 8.0.0

### PR DESCRIPTION
**Pull request by blackbaud-sky-build-user**
:robot: I have created a release *beep* *boop*
---

### Features

* update to Angular 15.2.9; trigger release for 8.0.0 ([#1355](https://github.com/blackbaud/skyux/issues/1355)) ([4b04139](https://github.com/blackbaud/skyux/commit/4b04139c0305ee0bc5b163a51749d6cec8dc283c))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1356/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/core-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/lists-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1356/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1356/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1356/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1356/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1356/playground/)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+* **components/forms:** update selected state background colors ([#1126](https://github.com/blackbaud/skyux/issues/1126)) ([8c820dd](https://github.com/blackbaud/skyux/commit/8c820dd58905ce533743b4eb790c891897d00a55))
+* **components/indicators:** announce wait component state changes to screen readers ([#968](https://github.com/blackbaud/skyux/issues/968)) ([a29983d](https://github.com/blackbaud/skyux/commit/a29983d36b759a86046d06401e8f25b555435743))
+* **components/indicators:** use alternate Font Awesome icon for default theme when available ([#1244](https://github.com/blackbaud/skyux/issues/1244)) ([ec8e9ae](https://github.com/blackbaud/skyux/commit/ec8e9ae7d7171ecc1df434d8e38c6b5ad5de48d8))
+* **components/indicators:** add ability to set aria controls and expanded states on the help inline component ([#1225](https://github.com/blackbaud/skyux/issues/1225)) ([#1230](https://github.com/blackbaud/skyux/issues/1230)) ([440c5bc](https://github.com/blackbaud/skyux/commit/440c5bc63b01d83d89b42f08e34d707a93a4a6e1))
```
Comment (johnhwhite) [inline CHANGELOG.md]: This is in 7 already.

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Good catch.

**Review by johnhwhite — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
:robot: Release is at https://github.com/blackbaud/skyux/releases/tag/8.0.0 :sunflower:

**Pull request merged**